.github/workflows/inspektor-gadget.yml

```diff
@@ -58,8 +107,42 @@ jobs:
                     if test -f $f; then echo $f; fi; \
                 done | xargs cp --parents --target-directory ~/cache-debs/
           fi
+    - name: Get inspektor-gadget from artifact.
+      uses: actions/download-artifact@v2
+      with:
+        name: inspektor-gadget-tar
+        path: /home/runner/work/inspektor-gadget/
+    - name: Build eBPF objects
+      run: |
+        tar -xf /home/runner/work/inspektor-gadget/inspektor-gadget.tar
+        cd inspektor-gadget
 
-    - name: Cache Go modules
+        make -C gadget-container ebpf-objects
+    - name: Lint
+      uses: golangci/golangci-lint-action@v2
+      with:
+        version: v1.43.0
+        working-directory: /home/runner/work/inspektor-gadget/inspektor-gadget/inspektor-gadget
+
+  build-kubectl-gadget:
+    name: Build kubectl gadget plugin for all platforms
```

Comment:
```suggestion
    name: Build kubectl-gadget
```
Currently, the name is truncated and we don't see the suffix (`amd64, linux`), see screenshot:

![image](https://user-images.githubusercontent.com/27575/151168824-9003f9e7-68f6-44f7-b0ee-1d265bd5f731.png)

Reply:
Done, thank you!